### PR TITLE
ELEMENTS-1316: Enhance nuxeo-data-table styles

### PR DESCRIPTION
**Pull request by pabgrall**
Submitting on behalf of a customer

### CODE REVIEW

File: ui/nuxeo-data-table/data-table-cell.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ import './data-table-templatizer-behavior.js';
             align-items: center;
             overflow: hidden;
             transition: flex-basis 200ms, flex-grow 200ms;
+            @apply --nuxeo-data-table-cell;
```
Comment (richardsd) [inline ui/nuxeo-data-table/data-table-cell.js:22]: To keep the naming consistent with other mixins (like [https://github.com/nuxeo/nuxeo-elements/blob/improve-ELEMENTS-1316-enhance-nuxeo-data-table-styling/ui/nuxeo-data-table/iron-data-table.js#L128](data table), I think this should be something like: 

```
@apply --iron-data-table-cell;
```
or 

```
@apply --data-table-cell;
```

File: ui/nuxeo-data-table/data-table-checkbox.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import '../widgets/nuxeo-checkmark.js';
             display: flex;
             align-items: center;
             justify-content: center;
+            @apply --nuxeo-data-table-checkbox;
```
Comment (richardsd) [inline ui/nuxeo-data-table/data-table-checkbox.js:20]: Same here, it should be something like:

```
@apply --iron-data-table-checkbox;
```
or
```
@apply --data-table-checkbox;
```

**Review by richardsd — CHANGES_REQUESTED** [review]
We need to first discuss and clarify why we are exposing a new mixin, and it should be renamed for consistency.

File: ui/nuxeo-data-table/data-table-cell.js (hunk 1 above)
Comment (Gabez0r) [reply ui/nuxeo-data-table/data-table-cell.js:22]: I believe `--iron-data-table-cell` is the best option here, for consistency with the other variables.

File: ui/nuxeo-data-table/data-table-checkbox.js (hunk 2 above)
Comment (Gabez0r) [reply ui/nuxeo-data-table/data-table-checkbox.js:20]: I believe `--iron-data-table-checkbox` is the best option here too.

**Review by richardsd — DISMISSED** [review]

**Review by Gabez0r — DISMISSED** [review]

**Comment by semisse** [comment]
Hello Patrick, can you please squash those two commits please?

**Review by semisse — CHANGES_REQUESTED** [review]
Please, squash the two commits into one to avoid redundant messages and to keep our codebase organized. Thank you Patrick.

**Review by richardsd — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Comment by semisse** [comment]
Thank you so much for the contribution, @pabgrall!

**Pull request merged**